### ISSUE TITLE
<input type="radio"> checked state not saved

### ISSUE THREAD

**Issue by zerodogg** [stereobooster/react-snap#84]
The state of radio input elements is not saved ot the generated html. React doesn't update it on hydrate() so for pages where there's a default checked radio button the form stays empty until the user clicks something. If there are multiple sets of radio buttons, when the user clicks one, react will also update the state of the other ones, so clicking any radio button suddenly displays the default values of the other radio buttons as well.

I'm using reactstrap, its input elements are using the checked property, which doesn't get written to the dom as attributes, but for the prerendered page it needs to be written out.

**Comment by stereobooster** [comment]
Solution would be the same as https://github.com/stereobooster/react-snap/pull/79. Walk through all radios and write out checked dom property as html attribute.

```js
Array.from(document.querySelectorAll('[type=radio]'))
  .forEach(x => { 
    if (x.checked) { 
      x.setAttribute('checked', 'checked')
    }
  });
```

**Comment by stereobooster** [comment]
fixed in 1.7.0

**Comment by zerodogg** [comment]
Fantastic, thank you very much!
